controlplane/pkg/tests/test_model.go

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2019 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tests
+
+import (
+	"context"
+
+	"github.com/networkservicemesh/networkservicemesh/controlplane/pkg/model"
+)
+
+type testModel struct {
+	model.Model
+	signalCh chan struct{}
+}
+
+func NewTestModel(signalChannel chan struct{}) *testModel {
```

Comment:
I think we can name it in a more representational way, for example - `healStartedCh` or something like that